frontend/src/App.vue

```diff
@@ -1,17 +1,28 @@
 <template>
   <div id="app">
     <img src="./assets/logo.png">
-    <hello></hello>
+    <input id="file" type="file" name="file" value="">
+    <button id="submit" type="button" @click="submit">送信</button>
   </div>
 </template>
-
 <script>
-import Hello from './components/Hello'
-
 export default {
   name: 'app',
-  components: {
-    Hello
+  methods: {
+    submit () {
+      const input = document.getElementById('file')
```

Comment:
Maybe it's better not to do `fetch` when no file is selected, something like this:

```js
const input = document.getElementById('file')
if (input.files.length === 0) return false;
...
```

Reply:
I understand. rewrite it